libs/datasets/timeseries.py

```diff
@@ -999,10 +1000,25 @@ def drop_new_case_outliers(
 
     zscores = grouped_df[CommonFields.NEW_CASES].apply(_calculate_modified_zscore)
 
+    new_tags = []
     to_exclude = (zscores > zscore_threshold) & (df_copy[CommonFields.NEW_CASES] > case_threshold)
-    df_copy.loc[to_exclude, CommonFields.NEW_CASES] = None
+    for idx, _ in to_exclude[to_exclude].iteritems():
```

Comment:
Should this be `df_copy[to_exclude]`?

Reply:
Yeah, that looks confusing. Either works. Added comment.